spec/models/wyeworker_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: wyeworkers
+#
+#  id         :integer          not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require "rails_helper"
+
+RSpec.describe Wyeworker, type: :model do
+  it "wires up properly as a source" do
+    source = create(:wyeworker_source)
+
+    expect(source.wyeworker_initiative_belongings.first.kind).to eql("source")
```

Comment:
I feel this is testing more the things that you did on factorybot than logic related to your program, might not be necessary 🤔 Anyways if it would be needed I think I'd write the test but with all the steps here.
I mean, something like:
- create a wyeworker
- create initiative
- link wyeworker as initiative's source
- validate

Reply:
There's lambdas and quite a bit of configuration involved, that's what I wanted to test. I see your point about being explicit, I changed the test accordingly!